src/components/DigitalCredentials/ChooseCredentials.vue

```diff
@@ -0,0 +1,14 @@
+<template>
+  <div class="mt-8">CHOOSE PLACEHOLDER</div>
```

Comment:
I recommend that the top-level element have an id. Same with all new components below.

Reply:
100%, will add them where missing 👍